contextvh/src/test/java/contextvh/translators/J2UpgradeTypeTest.java

```diff
@@ -24,23 +26,24 @@
 	/**
 	 * Translator of the J2Indicator to test.
 	 */
-	private J2UpgradeType translator;
+	private Translator translator = Translator.getInstance();
 	/**
 	 * The indicators to translate.
 	 */
 	private UpgradeType upgradeType;
 	private UpgradePair upgradePair;
 	private ArrayList<UpgradePair> a;
-
+	private List<UpgradePair> pairs;
 	/**
 	 * Initialise before every test.
 	 */
 	@Before
 	public void init() {
-		translator = new J2UpgradeType();
+		translator.registerJava2ParameterTranslator(new J2UpgradeType());
+		translator.registerJava2ParameterTranslator(new J2UpgradePair());
```

Comment:
Wouldn't it make more sense to mock the untested translator here? This way this test can fail if JUpgradePair fails. Doesn't really matter too much though.